tests/integration/sriov_test.py

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+import pytest
+
+import libnmstate
+from libnmstate.error import NmstateNotImplementedError
+from libnmstate.schema import Ethernet
+from libnmstate.schema import Interface
+from libnmstate.schema import InterfaceState
+from libnmstate.schema import InterfaceType
+from libnmstate.schema import SRIOV
+
+
+@pytest.mark.xfail(
```

Comment:
Hm! Neat. Does this throw you a warning resembling this one ? "UserWarning: Network state after test run does not match network state before test run:"

Reply:
No. But I think this warning is not related to your changes. You can check it using `pytest-args='-k <your_test_name>'` when executing run-test.sh

Reply:
@maiqueb it is unlikely that this test will expose this because `eth1` is cleaned up a lot of times. I guess for virtual interfaces, it will happen less likely when #593 is merged.

Reply:
tks @tyll .